<launch>

  <arg name="veh" default="$(env VEHICLE_NAME)" />

  <group ns="$(arg veh)">
    <node pkg="duckie_ctrl_node" name="duckie_ctrl_node" type="duckie_ctrl_node.py">
      <remap from="joy_mapper_node/joystick_override" to="/default/lane_controller_node/switch" />
      <remap from="lane_controller_node/car_cmd" to="/default/lane_controller_node/car_cmd" />
      <!-- <remap from="remote_steering_start" to="/modified/remote_steering_start" /> -->
      <!-- <remap from="lane_follow_override" to="/modified/lane_follow_override" /> -->
      <!-- <remap from="remote_steering_complete" to="/modified/remote_steering_complete" /> -->
    </node>
  </group>

</launch>
